Resources/Prototypes/_Moffstation/Entities/Clothing/Eyes/glasses.yml

```
@@ -51,3 +51,19 @@
    - type: Construction
      graph: MesonSunglasses
      node: sunglassesmeson
+
+- type: entity
+  parent: ClothingEyesHudDiagnostic
+  id: ClothingEyesGlassesDiagnosticSunglasses
+  name: diagnostic sunglasses
+  description: For those who have trouble with bright lights and careless cyborgs.
+  components:
+      - type: Sprite
+        sprite: _Moffstation/Clothing/Eyes/Glasses/diagsunglasses.rsi
+      - type: Clothing
+        sprite: _Moffstation/Clothing/Eyes/Glasses/diagsunglasses.rsi
+      - type: IdentityBlocker
+        coverage: EYES
+      - type: Construction
+        graph: DiagnosticSunglasses
+        node: sunglassesdiagnostic
```

Minor formatting nitpick
```suggestion
  components:
  - type: Sprite
    sprite: _Moffstation/Clothing/Eyes/Glasses/diagsunglasses.rsi
  - type: Clothing
    sprite: _Moffstation/Clothing/Eyes/Glasses/diagsunglasses.rsi
  - type: IdentityBlocker
    coverage: EYES
  - type: Construction
    graph: DiagnosticSunglasses
    node: sunglassesdiagnostic
```